README.md

```diff
@@ -1,4 +1,4 @@
-# Ed-Fi-Starter-Kit-SEA-Data-Collection
+# Starter-Kit-SEA-Modernization
```

Comment:
Include the additional contents I provided.

Reply:
I added